tools/dotcms-cli/cli/src/main/java/com/dotcms/cli/command/PushCommand.java

```diff
@@ -49,10 +50,20 @@ public class PushCommand implements Callable<Integer>, DotCommand {
     @Inject
     protected WorkspaceManager workspaceManager;
 
+
+    /**
+     * The Resolved DotPush command instances
+     * But this also allows for mocking push commands
+     * @return the resolved push commands or mocked instances
+     */
+    Iterable<DotPush> pushCommands(){
+        return CDI.current().select(DotPush.class);
```

Comment:
Rather than getting the CDI.current() when this is a bean, the regular pattern to get a list of all beans of a type is the following you can iterate over Instance
```
@Inject
    Instance<DotPush> pushCommands;
```

Reply:
yeah, But that allows me to mock that method and inject mocks so I can test the behavior. Actually this test was broken Im just fixing it

Reply:
There are other ways to inject mocks in this case for CDI and Quarkus but we can discuss that separately